app/src/main/res/layout/loginActivity/layout/title_layout.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+        android:id="@+id/display_title"
+        android:layout_width="wrap_content"
+        android:layout_height="match_parent"
+        android:layout_gravity="center"
+        android:gravity="center"
+        android:text="@string/app_title"
+        android:textColor="#CBD7F0"
```

Comment:
Maybe my description of the task is misleading you. But please change the text color to be #FFFFFF, which is consistent with PDA web version. Currently there is a color contrast warning for this.

Reply:
Also can you use referenced color from colors.xml? I assume that's the set of colors can be used throughout the App. Correct me is I'm wrong.

Reply:
ok , i will fix that